lib/src/Pos/Arbitrary/Block.hs

```diff
@@ -205,8 +207,8 @@ instance Arbitrary T.MsgGetBlocks where
     arbitrary = genericArbitrary
     shrink = genericShrink
 
-instance (Arbitrary (SscPayload ssc), Arbitrary (SscProof ssc), Bi Raw, SscHelpersClass ssc, HasConfiguration) =>
-         Arbitrary (T.MsgHeaders ssc) where
+instance (Arbitrary (SscPayload ssc), Arbitrary (SscProof ssc), Bi Raw, SscHelpersClass ssc, HasConfiguration, ssc ~ SscGodTossing) =>
```

Comment:
Are there any benefits in using `ssc ~ SscGodTossing` here?

Reply:
Nope, other than the fact that it already was in my clipboard.